integration/framework_test.go

```diff
@@ -248,3 +250,16 @@ func storeFileInCircleCI(f string) {
 		}
 	}
 }
+
+func laceworkIntegrationTestClient() (*api.Client, error) {
+	fmt.Println("Setting up host tests")
+	account := os.Getenv("CI_ACCOUNT")
+	key := os.Getenv("CI_API_KEY")
+	secret := os.Getenv("CI_API_SECRET")
+
+	lacework, err := api.NewClient(account, api.WithApiKeys(key, secret))
```

Comment:
Cloud we at least use API v2 for this?
```suggestion
	lacework, err := api.NewClient(account, api.WithApiV2(), api.WithApiKeys(key, secret))
```